sourceloop-lambda/authentication-service-lambda/cdk/lambda/common/stacks/redis.stack.ts

```diff
@@ -0,0 +1,117 @@
+import * as aws from '@cdktf/provider-aws';
+import { Fn, TerraformIterator, TerraformStack } from 'cdktf';
+import { Construct } from 'constructs';
+import { Redis } from '../../.gen/modules/redis';
+import { AwsProvider } from '../constructs/awsProvider';
+
+type Config = {
+  namespace: string;
+  environment: string;
+};
+
+export class RedisStack extends TerraformStack {
```

Comment:
I didn't intend for this to be a stand alone stack. There should be ElastiCache options that are part of our core stack for Lambdas that need to talk to EC.

Reply:
Elasticache is not mandatory for sourceloop auth service example here , The user just needs to change connector name and pass env vars if he wants to use elastic cache(have mentioned these things in the readme as well) , 
I will also create separate example to use elasticache in lambda as part of ARC-529